src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int side1;
+    private int side2;
+
+    public Rectangle(String color, int side1, int side2) {
+        super(color);
+        this.side1 = side1;
+        this.side2 = side2;
+    }
+
+    public int getArea() {
+        return side1 * side2;
+    }
+
+    @Override
```

Comment:
There should not be an empty line at the beginning of the method implementation according to the checklist.